server/src/modules/mentorship-response/mentorship-response.model.ts

```diff
@@ -11,6 +11,7 @@ import { MentorshipRequest } from '../mentorship-request/mentorship-request.mode
 export enum MentorshipResponseDecision {
   ACCEPTED = 'Accepted',
   DECLINED = 'Declined',
+  PENDING = 'Pending',
```

Comment:
I'm not sure what pending means: isn't every decision either accepted or declined?